design/annotation-for-power-cycling-and-deleting-failed-nodes.md

```diff
@@ -166,8 +205,16 @@ No changes are required to preserve the existing behaviour.
 
 ### Version Skew Strategy
 
-By shipping the new controller with the baremetal-operator that it consumes, we
-can prevent any version mismatches.
+CAPBM and BMO are delivered via the same image (both are installed by MAO, but running
```

Comment:
This information is all specific to OpenShift. The issue still exists, but you should rewrite this to not refer to the OpenShift-specific configuration or CAPBM. For example, "If CAPM3 and the BMO are in different pods, an upgrade or downgrade ..."

Reply:
done